### PR TITLE
Sei Lukso Celo deployment

### PR DESCRIPTION
**Pull request by 0xKurt**
**log**

- SEI
tx hash 0xde8f2134d16e1dcaa528af1092691d3de818d89fc5f47570ea9da128d4f0f8b4
gas used: 706_683
MultiRoundCheckout to: `0x710172b2C0aCc629A3FD23D436c347807dD5C412`

do you want to transfer ownership of the contract at 0x710172b2C0aCc629A3FD23D436c347807dD5C412 **(MRC)** to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0xac677f0d4d106f03e8eae903fcf8a4decfe99258c6fa5ae0171245fd1a059733
gas used: 35_807

do you want to transfer ownership of the contract at 0x1E18cdce56B3754c4Dca34CB3a7439C24E8363de **(Admin)** to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0xc73ebee83e471abfb0dd8904e0868c180a9d2572177a966c9e00626d37b1531d
gas used: 28_603

**note: cannot verify implementation?!**

### CODE REVIEW

**Comment by 0xKurt** [comment]
chainId: 42220
network: celo (from ethers: celo)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 169_980_721_050_000_000_000
do you want to deploy the MultiRoundCheckout contract? [y/n]: y

deploying...
tx hash 0xb7592d512be7e70089b324f78104c494495d4fd515dfe067c07700cbac4fe679
gas used: 706_433
MultiRoundCheckout to: 0x7eD4634F6c399b2Ab35C8973827D66d52C00dfD2

chainId: 42220
network: celo (from ethers: celo)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 169_951_340_900_000_000_000
do you want to transfer ownership of the contract at 0x7eD4634F6c399b2Ab35C8973827D66d52C00dfD2 to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0xc1f7dfd521fb0e539bd9eac695a4c55cf2905e6e8b15def6bde80907ac707fc1
gas used: 35_807

chainId: 42220
network: celo (from ethers: celo)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 169_950_982_830_000_000_000
do you want to transfer ownership of the contract at 0x91e20A45A8A324fE17F77a0C610ca2553079e449 to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0xb4e4dc2e6be64682b70637d61171b71a719e745d5362445f4186ab2999d9081b
gas used: 28_603

**Comment by 0xKurt** [comment]
chainId: 44787
network: celoAlfajores (from ethers: celoAlfajores)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 3_035_489_180_000_000_000
do you want to deploy the MultiRoundCheckout contract? [y/n]: y

deploying...
tx hash 0xefa06937ea2d0f2fc31fa7d74167ac97e4dbcbf31b48377ae8bb6906366a066d
gas used: 706_433
MultiRoundCheckout to: 0x313eC6CA225C40Bc670d8cd4b063734BD22ad1ab

chainId: 44787
network: celoAlfajores (from ethers: celoAlfajores)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 3_006_109_030_000_000_000
do you want to transfer ownership of the contract at 0x313eC6CA225C40Bc670d8cd4b063734BD22ad1ab to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0xdd9f5ae2d1543a62069e045d1367ab8bd98c61bf9e84a0fb695e3f24044beb57
gas used: 35_807

chainId: 44787
network: celoAlfajores (from ethers: celoAlfajores)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 3_005_750_960_000_000_000
do you want to transfer ownership of the contract at 0xf5cA96151d1a9998d234963433bfd3f6feC7aAc2 to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0x0b6af6ac89a8941143607af06487bf777250e38adbfed0fa55f40a1ccb0ecac3

**Comment by 0xKurt** [comment]
chainId: 4201
network: luksoTestnet (from ethers: luksoTestnet)
account: 0xE7eB5D2b5b188777df902e89c54570E7Ef4F59CE
balance: 9_997_632_974_488_953_881
do you want to deploy the MultiRoundCheckout contract? [y/n]: y

deploying...
tx hash 0xbdade70c50f2d6e964787acd624298b14ac3efce3d102faa70d71fc768087e02
gas used: 706_683
MultiRoundCheckout to: 0x606BfaCCc19F040F55C7be25C1Db02a4201bf8f0

chainId: 4201
network: luksoTestnet (from ethers: luksoTestnet)
account: 0xE7eB5D2b5b188777df902e89c54570E7Ef4F59CE
balance: 9_997_339_085_968_381_686
do you want to transfer ownership of the contract at 0x606BfaCCc19F040F55C7be25C1Db02a4201bf8f0 to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0xE7eB5D2b5b188777df902e89c54570E7Ef4F59CE
tx hash 0xafc3935ab56a66a256967064c20e9fcc080d2081c8dad77fff2b99bc4c312954
gas used: 35_807

chainId: 4201
network: luksoTestnet (from ethers: luksoTestnet)
account: 0xE7eB5D2b5b188777df902e89c54570E7Ef4F59CE
balance: 9_997_335_505_268_131_037
do you want to transfer ownership of the contract at 0x2447dD8C1f4cd4361a649564Bd441787edf8c03A to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0xE7eB5D2b5b188777df902e89c54570E7Ef4F59CE
tx hash 0x0374db6492d10ef0db3497fe586a5f8a2c07a04014f42fd696baa74c363351d7
gas used: 28_603

File: scripts/deploy/transferOwnership.ts [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { getEnv } from "../../lib/utils";
 // Script to transfer ownership of Ownable contracts.
 // It can be used both for ProxyAdmin and MultiRoundCheckout contracts.
 
-const CONTRACT_ADDRESS = "";
-const NEW_OWNER_ADDRESS = "";
+const CONTRACT_ADDRESS = "0x2447dD8C1f4cd4361a649564Bd441787edf8c03A";
```
Comment (thelostone-mc) [inline scripts/deploy/transferOwnership.ts]:
```suggestion
const CONTRACT_ADDRESS = "";
```

File: scripts/deploy/transferOwnership.ts [hunk 2]
```diff
@@ -5,8 +5,8 @@ import { getEnv } from "../../lib/utils";
...
+const NEW_OWNER_ADDRESS = "0x79427367e9Be16353336D230De3031D489b1b3c3";
```
Comment (thelostone-mc) [inline scripts/deploy/transferOwnership.ts]:
```suggestion
const NEW_OWNER_ADDRESS = "";
```

**Comment by 0xKurt** [comment]
chainId: 42
network: lukso (from ethers: lukso)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 76_277_652_225_628_864_517
do you want to deploy the MultiRoundCheckout contract? [y/n]: y

deploying...
tx hash 0xfb6a6d95e2e69302dce1a6781630bf16eba51c91ed52ffd8b81e212b18d298d5
gas used: 706_414
MultiRoundCheckout to: 0x029dFAf686DfA0efdace5132ba422e9279D50b5b

0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
chainId: 42
network: lukso (from ethers: lukso)
account: 0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
balance: 76_274_340_035_469_710_918
do you want to transfer ownership of the contract at 0x029dFAf686DfA0efdace5132ba422e9279D50b5b to address 0x79427367e9Be16353336D230De3031D489b1b3c3? [y/n]: y

init...
current owner  0x8C180840fcBb90CE8464B4eCd12ab0f840c6647C
tx hash 0x5b4ec6cefad20f8d57c3ae88f3ca02dc4e912349f625b0144e07815230b1591a
gas used: 35_807

**Pull request closed without merging**